manuscript/manuscript.Rmd

```diff
@@ -116,45 +116,44 @@ What do we mean by an "agent"?
-For instance, we might represent a ecological community of 10 interacting species using an environment with 10 continuously valued states representing the possible population densities of each species.
-The action space may similarly complex, and the response of the environment to any possible action may be deterministic or stochastic in nature.
-The reward will in general depend on both the current state and choice of actions.
-An environment must also specify a stopping condition, such as when a particular state is reached or a pre-defined interval of time has elapsed.
-In some environments, a reward is only received after the agent successfully completes a potentially long sequence of complex actions, such as solving a maze.
-In other environments, an agent may have to learn to forgo immediate reward (or even accept an initial negative reward or cost) in order to maximize the net discounted reward over the simulation period, as we illustrate in examples here.
+An environment is a mathematical function, computer program, or real world experience that takes an agent's proposed *action* as input and returns an observation of its current *state* and an associated *reward* as output.
+<!-- ML: what are classical approaches below here? -->
+In contrast to classical approaches, there are few restrictions on what comprises a state or action. 
+The state may be continuous or discrete, completely or partially observed, single or multidimensional.
+For instance, we might represent an ecological community of 10 interacting species with a 10-dimensional continuously valued state space signifying the population density of each species.
+The action space can be similarly complex, since an action can be continuous or discrete, single or multidimensional.
+For the previous environment, we might set the action space to be continuously valued and 10-dimensional, where each dimension corresponds to the amount of conservation effort aimed at the different species over a time step.   
+The main focus of an RL environment, however, is in the environment's transition dynamics and reward function.
+The designer of the environment can make the environment follow any transition and reward function provided that both are functions of the current state and action.
+This ability allows RL environments to model a broad range of decision making problems.
+For example, we can set the transitions to be deterministic or stochastic.
+We can also specify the reward function to be *sparse*, whereby a positive reward is only received for state-action pairs that can only be visited after a long sequence of actions, e.g. the end point in a maze.
+In other environments, an agent may have to learn to forgo immediate rewards (or even accept an initial negative reward) in order to maximize the net discounted reward over the simulation period, as we illustrate in examples here.
+Another important consideration in RL is whether to use an episodic or non-episodic environment.
```

Comment:
I think episodic environments are nuance we can leave for the appendix.  drawing attention to the fact that we're focused on episodic environments here is just asking for criticism (policy-makers may love policy deadlines like 30% by 2030, but does conservation really have finite timelines?).  But I think discussing finite/infinite timelines, the "right" discount rate etc is outside of our scope, since the issue isn't unique to RL and has been discussed before.  I think you can just cut lines 152-155